### DISCUSSION TITLE
Emulate a mouse with start on a controller connected directly to the PC (or USB/IP) during a stream / Adding an overlay on the sunshine side

### DISCUSSION THREAD

**Discussion by moi952** [LizardByte/community#471] [Sunshine]
I wonder if it would not be more relevant to have an overlay on the sunshine side rather than on the client side.

This would allow similar functionalities during streams regardless of the client, we could display the overlay even from a controller or keyboard directly connected to the PC.

Currently I use my PS5 controller connected to the PC (via virtualHere) but I cannot emulate the mouse with it which restricts the use quite a bit.

It would be great if a development on the sunshine side was done first to implement an overlay which would enable mouse emulation, then in the future other features like these:
- activate the display of statistics on the client
- activate mouse emulation
- launch keyboard shortcuts that we could create / configure in sunshine and other things

Perhaps in the future we could only use this overlay, which would allow all the functionalities to be grouped together on the sunshine side 🙂

A bit like the geforce now overlay

**Comment by ReenigneArcher** [comment]
I like the idea.
